trade-route-example-config.json

```diff
@@ -0,0 +1,39 @@
+{
+  "goerli": {
+    "type": "collateral",
+    "token": "0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6",
+    "owner": "0x5bA371aeA18734Cb7195650aFdfCa4f9251aa513",
+    "mailbox": "0xCC737a94FecaeC165AbCf12dED095BB13F037685",
+    "interchainGasPaymaster": "0xf857706CE59Cb7AE6df81Bbd0B0a656dB3e6beDA"
```

Comment:
Would be cool if dev didn't have to specify this and it was retrieved from core artifacts instead

Reply:
That's what we had originally before, but the goal here was to support configuration from the user side and JSON seemed preferable to a typescript file

Reply:
I think @jmrossy 's point is we can pull this from the sdk package

Reply:
Yeah but my intention was for this to be a JSON file and not a Typescript file (the former obviously can't import).

You could make the script just fill in the gaps I guess, but I was (possibly prematurely) anticipating a permissionless deployment world where these contract addresses may not be retrievable from the SDK. I personally think that this is fine enough for folks to handle